botMessages.ts

```diff
@@ -15,13 +15,14 @@ I wait in an enforced slumber biding my time,
 until the fated moment when I shall be set free once again, 
 to reignite with my reminders upon your unorganized world! 🧞`;
 
-MESSAGES.SUBSCRIBED_FAIL = `Chat already susbscribed to /remiders`
+MESSAGES.SUBSCRIBED_FAIL = `Chat already susbscribed to /remiders`;
 
-MESSAGES.UNSUBSCRIBED_FAIL = `Your chat is not even on my list, bby 🧞`
+MESSAGES.UNSUBSCRIBED_FAIL = `Your chat is not even on my list, bby 🧞`;
 
-MESSAGES.REMINDERS_CLEANING_PLAN = `🧹🪥 This is an automated Putzplan reminder! 🧼🫧`
+MESSAGES.REMINDERS_CLEANING_PLAN_EVEN_WEEK = `🧹🪥 This is an automated Putzplan reminder! Left nothing behind! 🧼🫧`;
+MESSAGES.REMINDERS_CLEANING_PLAN = `🧹🪥 This is an automated Putzplan reminder! Shine right like a diamond 🧼🫧`;
```

Comment:
can we make the other week `ODD_WEEk`

Reply:
`MESSAGES.REMINDERS_CLEANING_PLAN` = 'lk;asjdlk'
`EVEN_WEEK` = 'saldk'
...
`MESSAGES.REMINDERS_CLEANING_PLAN` + `EVEN_WEEK`

Reply:
hm i wouldn't generalize it that much, its easyer to read and to change it, if we let it separate like this.
But if you really want it I change it darling  :*